src/pages/LogInWithShortLivedAuthTokenPage.js

```diff
@@ -105,9 +121,7 @@ LogInWithShortLivedAuthTokenPage.propTypes = propTypes;
 LogInWithShortLivedAuthTokenPage.defaultProps = defaultProps;
 LogInWithShortLivedAuthTokenPage.displayName = 'LogInWithShortLivedAuthTokenPage';
 
-export default compose(
-    withLocalize,
-    withOnyx({
-        account: {key: ONYXKEYS.ACCOUNT},
-    }),
-)(LogInWithShortLivedAuthTokenPage);
+export default withOnyx({
+    account: {key: ONYXKEYS.ACCOUNT},
+    session: {key: ONYXKEYS.SESSION},
```

Comment:
The `session` prop appears unused. Do you know if it's needed here?

Reply:
We can remove it most probably. Seems like we removed the usage, but forgot to remove this key here. /cc @NikkiWines to be double sure

Reply:
Ah yes, we can remove this 👍 I  can do that as part of https://github.com/Expensify/App/pull/29526 unless you'd like to make a separate PR fro it @roryabraham

Reply:
@NikkiWines if you want to do it as part of https://github.com/Expensify/App/pull/29526 that would be great!

Reply:
[Done](https://github.com/Expensify/App/pull/29526/commits/5c8519f4483f40629ac77065be39197cf1b47186)!